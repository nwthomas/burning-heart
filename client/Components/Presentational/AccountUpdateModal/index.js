export { default as AccountUpdateModal } from "./AccountUpdateModal";
