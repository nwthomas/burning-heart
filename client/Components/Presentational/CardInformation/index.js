export { default as CardInformation } from "./CardInformation";
