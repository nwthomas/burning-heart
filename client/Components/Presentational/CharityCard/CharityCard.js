import React from "react";
import { Text, View, StyleSheet } from "react-native";
import { Link } from "react-router-native";

const CharityCard = ({ charity }) => {
  return (
    <Link to={`/search/profile/${charity.id}`}>
      <View style={styles.cardContainer}>
        <Text style={styles.charityCardText}>{charity.charityName}</Text>
      </View>
    </Link>
  );
};

const styles = StyleSheet.create({
  cardContainer: {
    alignSelf: "stretch",
    paddingTop: 30,
    paddingBottom: 30,
    borderBottomColor: "#cacacf",
    borderBottomWidth: 1
  },
  charityCardText: {
    fontFamily: "Roboto-Medium",
    fontSize: 16
  }
});

export default CharityCard;
