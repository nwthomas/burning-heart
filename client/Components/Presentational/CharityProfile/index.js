export { default as CharityProfile } from "./CharityProfile";
