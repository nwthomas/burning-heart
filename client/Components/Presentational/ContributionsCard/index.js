export { default as ContributionsCard } from "./ContributionsCard";
