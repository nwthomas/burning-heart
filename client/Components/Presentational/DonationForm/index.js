export { default as DonationForm } from "./DonationForm";
