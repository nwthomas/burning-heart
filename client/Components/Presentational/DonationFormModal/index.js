export { default as DonationFormModal } from "./DonationFormModal";
