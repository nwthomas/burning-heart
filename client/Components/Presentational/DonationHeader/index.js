export { default as DonationHeader } from "./DonationHeader";
