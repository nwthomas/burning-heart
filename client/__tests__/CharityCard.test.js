import React from "react";
import "react-native";
import { CharityCard } from "../Components/Presentational/CharityCard";
import renderer from "react-test-renderer";

describe("<CharityCard />", () => {
  it("should render the component without crashing", () => {
    const charity = { charityName: "name" };
    const helpers = renderer.create(<CharityCard charity={charity} />);
  });

  it("should match the snapshot of the tree", () => {
    const charity = { charityName: "name" };
    const tree = renderer.create(<CharityCard charity={charity} />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
