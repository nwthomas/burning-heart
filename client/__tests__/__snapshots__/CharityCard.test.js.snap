// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CharityCard /> should match the snapshot of the tree 1`] = `
<View
  onPress={[Function]}
  style={
    Object {
      "alignSelf": "stretch",
      "borderBottomColor": "#cacacf",
      "borderBottomWidth": 1,
      "paddingBottom": 30,
      "paddingTop": 30,
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "Roboto-Medium",
        "fontSize": 16,
      }
    }
  >
    name
  </Text>
</View>
`;
